includes/Modules/AdSense.php

```diff
@@ -435,8 +435,9 @@ protected function create_data_request( Data_Request $data ) {
 			case 'GET:account-url':
 				return function() {
 					$account_id = $this->get_data( 'account-id' );
-					if ( ! is_wp_error( $account_id ) && $account_id ) {
-						return sprintf( 'https://www.google.com/adsense/new/%s/home', $account_id );
+					if ( ! is_wp_error( $account_id ) && $account_id && $this->authentication->profile()->has() ) {
+						$profile_email = $this->authentication->profile()->get()['email'];
+						return sprintf( 'https://www.google.com/adsense/new/u/%1$s/%2$s/home', $profile_email, $account_id );
```

Comment:
This should also use the `authuser` query param instead of the `/u/EMAIL` path segment.

Reply:
Updated.